File: app/src/androidTest/java/app/we/go/emojidraw/TestApplication.java
package app.we.go.emojidraw;

import app.we.go.emojidraw.arch.di.ApplicationComponent;
import app.we.go.emojidraw.arch.di.DaggerTestApplicationComponent;

public class TestApplication extends ThisApplication {

    @Override
    protected ApplicationComponent createComponent() {
        return DaggerTestApplicationComponent.builder()
                .build();

    }
}

File: app/src/androidTest/java/app/we/go/emojidraw/TestApplication.kt
package app.we.go.emojidraw

import app.we.go.emojidraw.arch.di.ApplicationComponent
import app.we.go.emojidraw.arch.di.DaggerTestApplicationComponent

class TestApplication : ThisApplication() {

    override fun createComponent(): ApplicationComponent {
        return DaggerTestApplicationComponent.builder()
                .build()

    }
}
